app/Filament/Admin/Resources/EggResource/Pages/CreateEgg.php

```diff
@@ -199,7 +199,11 @@ public function form(Form $form): Form
                                             '*' => trans('admin/egg.error_reserved'),
                                         ])
                                         ->required(),
-                                    TextInput::make('default_value')->label(trans('admin/egg.default_value'))->maxLength(255),
+                                    TextInput::make('default_value')
+                                        ->label(trans('admin/egg.default_value'))
+                                        ->required(fn (Get $get) => in_array('required', $get('rules')))
```

Comment:
Not sure if the default value should be required. Sometimes you only want it to be required when creating a server but not on the egg itself.

What do you think, @QuintenQVD0 ?

Reply:
I do agree. For example for password fields.